LibreNMS/OS/Routeros.php

```diff
@@ -378,7 +378,7 @@ public function discoverWirelessDistance()
                 '60G: Sta > ' . $entry['mtxrWl60GStaRemote'],
                 $entry['mtxrWl60GStaDistance'],
                 1,
-                1000
+                100000
```

Comment:
Interesting, considering the MIB says this is defined in meters not centimeters :D

Reply:
The graph shows kilometres and not metres, so the MIB is saying it wrong. The connection is not really 18 km.

Before changes:
![image](https://user-images.githubusercontent.com/2546041/188497994-d65ceb45-6170-41ca-aad6-47c23ab78619.png)

After changes:
![image](https://user-images.githubusercontent.com/2546041/188498094-51c2e7d7-9459-4f8b-86f6-3f8c228f5901.png)

Reply:
Yes, the graph data is scaled to km for consistency.

Speaking of consistency, do all devices/firmwares report this value the same?

Reply:
> Yes, the graph data is scaled to km for consistency.

I understand, that's not a problem.

> 
> Speaking of consistency, do all devices/firmwares report this value the same?

Tested on the first firmware with 60G support, the middle and the latest, the same value is returned everywhere. Also tested Cube 60G, WAP 60G, RBLHG 60G (no longer sold) and the same values are returned everywhere.

Reply:
Thanks for confirming.   Mikrotik isn't the best at snmp (but also not the worst)